docs/sphinx/source/whatsnew/v0.11.2.rst

```diff
@@ -25,6 +25,8 @@ Documentation
 ~~~~~~~~~~~~~
 * Edited docstrings for :py:func:`~pvlib.pvsystem.dc_ohms_from_percent` and
   :py:func:`~pvlib.pvsystem.dc_ohmic_losses` for clarity. (:issue:`1601`, :pull:`2229`)
+* Added variables `spectra` and `spectra_components` to the
+  :ref:`nomenclature` page. (:issue:`2150`, :pull:`2264`)
```

Comment:
I'm -0.5 to a whatsnew note for new Nomenclature entries.  If we do retain this note, it should be moved down in the list to be after the note about creating the Nomenclature page itself.

Reply:
My motivation for including it was more related to making users aware of this (hopefully) standard variable name rather than the fact that the nomenclature (something inside me still wants to type glossary 😂) page being updated.
Do you think that is important?
If not, I think it's fine to remove this entry. If it is retained then +1 to moving it down

Reply:
I think one note about adding terms is fine. Several notes (one for each PR) may be clutter.